components/_expanding-components/uncheck-radio-button.js

```diff
@@ -0,0 +1,37 @@
+export const UncheckRadioButton = function(elementClass) {
+  const expandingComponent = elementClass
+  let expandingElement
+
+  const cacheDom = function() {
+    expandingElement = document.querySelectorAll('.' + expandingComponent)
+  }
+
+  const bindEvents = function() {
+    expandingElement.forEach(labelElement => {
+      labelElement.addEventListener('click', () => findInputForLabel(labelElement))
+    })
+  }
+
+  const findInputForLabel = function(labelElement) {
+    const labelElementFor = labelElement.htmlFor
+    const inputId = labelElementFor
+
+    if (labelElementFor) {
+      uncheckRadioButton(inputId)
+    }
+  }
+
+  const uncheckRadioButton = function(inputId) {
```

Comment:
you have a method with the same name  as the compontent class. is this a constructor?

Reply:
its not a constructor but the component class has one function, to uncheck radio buttons for expanding components, I can't think of a better name for the class or the method, do you have any ideas

Reply:
I guess for me, the entire file here sounds more like a couple of methods too add on the expander components.

Reply:
so you are saying I should not go DRY but instead I should repeat myself?